stock_picking_batch_extended/migrations/15.0.3.0.0/pre-migration.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+def _set_move_id_picking_batch(env):
+    """Pre-fill column"""
+    openupgrade.logged_query(
```

Comment:
This script is not needed, as now related fields are pre-filled with a similar SQL to this one.

Reply:
Ok, I just removed it :)